src/windows/functions.cpp

```diff
@@ -393,6 +393,37 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterEjectable(TCHAR driveLetter) {
+  TCHAR devicePath[8];
+  wsprintf(devicePath, TEXT("%c:\\"), driveLetter);
+
+  MountUtilsLog("Checking whether drive is ejectable: " + std::string(1, driveLetter));
```

Comment:
Do you also want to make the other `MountUtilsLog` functions report the drive letter they're operating on?